index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio: setup and mobile version skeleton</title>
+</head>
+<body>
+
+    
+         <header>
+            <a id="logo">My Logo</a><br><br>
+        <nav class="manu">
+            <img id="img2" src="https://icon-library.com/images/menu-icon-png-3-lines/menu-icon-png-3-lines-10.jpg" alt="#" class="menu">
+    </nav>
```

Comment:
- [ ] **(Optional)** Nice job implementing the required changes 👏. I noticed that you can do one more improvement. In the picture below, the color of the hamburger button in your project is `black` in color while that used in the Figma design template is `white`.
### Your Portfolio

![image](https://user-images.githubusercontent.com/77106962/143483724-7484696b-9150-4553-8c72-94e7fb881bd7.png)
### Figma's Template

![image](https://user-images.githubusercontent.com/77106962/143483824-e0df9849-a81d-497f-9042-2a7509d948c6.png)

### **I strongly advise that you implement this optional change** 🙏